### PR TITLE
#218: added PR template

### PR DESCRIPTION
**Pull request by commjoen**
Covering #218 to have a PR.

### CODE REVIEW

**Comment by sushi2k** [comment]
Great addition. LGTM

**Comment by OMTGreleaser** [comment]
Results for commit cc09a8a2767eb3da0c188e9181f569d950aafc94: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

**Comment by OMTGreleaser** [comment]
Results for commit 89fdcead106cbda1bf99820abb28fe47cc199e30: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:
+
+- [ ] Your PR is linked to an issue.
+- [ ] Your PR follows the [contribution guidelines](https:// "Contribution guidelines")
+- [ ] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
+- [ ] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?
+
+If your PR is related to an issue. Please end your PR test with the following line:
```
Comment (TheDauntless) [inline PULL_REQUEST_TEMPLATE.md]: Would just remove this line, because it's a requirement.

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:
+
+- [ ] Your PR is linked to an issue.
+- [ ] Your PR follows the [contribution guidelines](https:// "Contribution guidelines")
```
Comment (TheDauntless) [inline PULL_REQUEST_TEMPLATE.md]: Update the link (the file didn't exist when I first wrote this ;) )

**Pull request merged**